src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,14 +1,34 @@
 package core.basesyntax;
 
-/**
- * <p>Дано файл, потрібно прочитати його вміст і вибрати всі слова що починаються на `w`.
- * Результат повернути у вигляді відсортованого масиву (за замовчуванням). Всі слова повинні
- * бути в нижньому регістрі. У випадку якщо таких слів не знайдено повернути пустий масив.
- * Приклад: Width world Wide web
- * Результат: web wide width world</p>
- */
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class FileWork {
+    private static final String KEY = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> result = new ArrayList<>();
+        try {
+            List<String> lines = Files.readAllLines(Paths.get(fileName));
+            for (String line : lines) {
+                line = line.toLowerCase();
+                line = line.replaceAll("[^a-z]", " ");
+                String[] separateWords = line.split("\\s+");
+                for (String word : separateWords) {
+                    if (word.startsWith(KEY)) {
+                        result.add(word);
+                    }
+                }
+            }
+        } catch (IOException e) {
+            System.out.println("No such a file");
```

Comment:
Don’t ignore exceptions. Re-throw them as RuntimeException(e.g. throw new RuntimeException(e)) if you don’t expect your code to fail here, or make necessary actions at the catch block to fix the problem.
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_05.html